lib/streamlit/__init__.py

```diff
@@ -265,3 +265,8 @@ def _update_logger() -> None:
 experimental_connection = _deprecate_func_name(
     connection, "experimental_connection", "2024-04-01", name_override="connection"
 )
+
+# make it possible to call streamlit.components.v1.html etc. by importing it here
+# import in the very end to avoid partially-initialized module import errors, because
+# streamlit.components.v1 also uses some streamlit imports
+import streamlit.components.v1
```

Comment:
nit: I think you need to add a `# noqa: F401` here

Reply:
done!